tests/Spec/Utils.hs

```diff
@@ -48,7 +48,7 @@ matchesTextShowSpec _ = prop "TextShow instance" (prop_matchesTextShow :: Int ->
 -- | Verifies that a type's 'Show' instances coincide for both 'String's and 'Text',
 -- irrespective of precedence.
 prop_matchesTextShow :: (Show a, TextShow a) => Int -> a -> Expectation
-prop_matchesTextShow p x = fromString (showsPrec p x "") `shouldBe` showbPrec p x
+prop_matchesTextShow p x = showbPrec p x `shouldBe` fromString (showsPrec p x "")
```

Comment:
Right part should be the expected.

Reply:
Yes, good point. Do you mind changing the order for `prop_matchesTextShow1` and `prop_matchesTextShow2` as well while you're at it?